Manifests/ManagedPreferencesApplications/com.microsoft.autoupdate2.plist

```diff
@@ -432,6 +433,26 @@ Writes logs to: /Library/Logs/Microsoft/autoupdate.log</string>
 			<key>pfm_value_unit</key>
 			<string>Days</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>4.48</string>
+			<key>pfm_default</key>
+			<integer>60</integer>
+			<key>pfm_description</key>
+			<string>How many minutes is the final countdown timer once the update deadline is reached.</string>
+			<key>pfm_description_reference</key>
+			<string>IT admins can now override the default deadline time of 60 minutes through the UpdateDeadline.FinalCountDown preference</string>
+			<key>pfm_documentation_url</key>
+			<string>https://docs.microsoft.com/en-us/officeupdates/release-history-microsoft-autoupdate#june-2022-release</string>
+			<key>pfm_name</key>
+			<string>UpdateDeadline.FinalCountDown</string>
+			<key>pfm_title</key>
+			<string>Update Deadline: Final Countdown Timer</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+			<key>pfm_value_unit</key>
+			<string>Minutes</string>
```

Comment:
nit, but usually set unit values to be all lowercase

Reply:
The others in this file are capitalized, e.g. Days, so I stuck with the matching format.